mri/test/test_optimizer.py

```diff
@@ -17,6 +17,7 @@
 from mri.reconstruct.fourier import FFT2, NFFT
 from mri.numerics.reconstruct import sparse_rec_fista
 from mri.numerics.reconstruct import sparse_rec_condatvu
+from mri.numerics.reconstruct import sparse_rec_pogm
```

Comment:
Maybe we can have all these imports on the same line.

Reply:
Well, PEP8 would still need to move to a different line, this just looks cleaner to me. Let me know if this needs to be changed still, I am fine with it also

Reply:
Well I planned to move to that itself, just didnt know how clean it was. Will get an update

Reply:
You didn't look at the SO answer though haha, nv mind it's ok like this